app/tests/unit/pages/project-revision/[projectRevision]/form/managers.test.tsx

```diff
@@ -327,19 +315,15 @@ describe("The Project Managers form page", () => {
       });
 
     pageTestingHelper.loadQuery();
-    pageTestingHelper.renderPage();
+    const mockPush = jest.fn();
+    pageTestingHelper.renderPageWithMockRouter({ push: mockPush });
     handleSubmit();
     expect(mockPush).toHaveBeenCalledWith(
       getProjectRevisionPageRoute("mock-proj-rev-2")
     );
```

Comment:
I was thinking we could expose the mock router object on the helper, similarly to what we do for the relay environment. This way the user wouldn't have to setup their own mock functions and configure the helper, it would all be done automatically, and the API would look like this:

```ts
pageTestingHelper.renderPage();
expect(pageTestingHelper.router.push).toHaveBeenCalledWith(...)
```
I'm not even sure there would be value in \*not\* mocking the router